src/mpi/datatype/pack_external.c

```diff
@@ -120,7 +120,7 @@ int MPI_Pack_external(const char datarep[],
         goto fn_fail;
     }
     /* --END ERROR HANDLING-- */
-    mpi_errno = MPIR_Segment_init(inbuf, incount, datatype, segp, 1);
+    mpi_errno = MPIR_Segment_init(inbuf, incount, datatype, segp, 0);
```

Comment:
If it's unused, it should be removed instead of just changing the passed parameter.